File: Moka/DTOs/StoreDto.cs
namespace Moka.DTOs
{
    public class StoreDto
    {
        public int Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string? SapCode { get; set; }
        public string? TradeName { get; set; }
        public string? Address { get; set; }
        public string? PostalCode { get; set; }
        public string? City { get; set; }
        public string? TaxId { get; set; }
        public string? Notes { get; set; }
    }
}

File: moka-web/src/features/stores/types/StoreDto.ts

export interface StoreDto{
    id: number;
    name: string;
    sapCode?: string;
    tradeName?: string;
    address?: string;
    postalCode?: string;
    city?: string;
    taxId?: string;
    notes?: string;

}

export interface StoreApiResponse {
    totalItems: number;
    page: number;
    pageSize: number;
    items: StoreDto[];
}
 
export interface CreateStoreDto{
    name: string;
    sapCode?: string;
    tradeName?: string;
    address?: string;
    postalCode?: string;
    city?: string;
    taxId?: string;
    notes?: string;
}
 
export interface UpdateStoreDto{
    name: string;
    sapCode?: string;
    tradeName?: string;
    address?: string;
    postalCode?: string;
    city?: string;
    taxId?: string;
    notes?: string;
}